core/unit_test/TestTeamBasic.hpp

```diff
@@ -65,23 +65,24 @@ TEST(TEST_CATEGORY, team_for) {
       1000);
 }
 
-// FIXME_OPENMPTARGET wrong results
-#ifndef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
I don't like this change, since it will look like the test is actually running for OpenMPTarget while it actually isn't.